Devscord.DiscordFramework/Integration/Services/DiscordClientChannelsService.cs

```diff
@@ -109,7 +110,19 @@ public async Task<IEnumerable<Message>> GetMessages(DiscordServerContext server,
                 contexts.SetContext(user);
 
                 var commandParser = new CommandParser();
-                var request = commandParser.Parse(message.Content, message.Timestamp.UtcDateTime);
+                DiscordRequest request;
+                try
+                {
+                    request = commandParser.Parse(message.Content, message.Timestamp.UtcDateTime);
+                }
+                catch // should almost never go to catch block, but in rare cases Parse() can throw an exception
```

Comment:
jakie to są wyjątki? może powinniśmy zmienić coś w samym parserze, a nie na zewnątrz

Reply:
zostawię tak, bo CommandParser może rzucać wyjątek InvalidArguments